app/integration/component-proxy/src/main/java/io/syndesis/integration/component/proxy/ComponentCustomizer.java

```diff
@@ -44,23 +44,37 @@ default void consumeOption(Map<String, Object> options, String name, Consumer<Ob
         }
     }
 
+    default void consumeStringOption(Map<String, Object> options, String name, Consumer<String> consumer) {
+        Object val = options.remove(name);
+        if (val != null) {
+            consumer.accept(String.valueOf(val));
```

Comment:
So this would bypass any custom type converter registered for translating to a string, so I guess it has only to be used when you know that the type is a string right ? if so it it is worth documenting it

Reply:
It is called `consumeStringOption` and it accepts `Consumer<String>`, kinda apparent to me the the caller is asking for a `String`, no?

Reply:
I mean that it does not involve the type converter so the result may be unexpected

Reply:
Aah, that bit, got it, yeah, delegating to `consumeOption` might be better...